manifests/init.pp

```diff
@@ -247,6 +247,11 @@
 #   Default: false
 #   Replicate the registry Nginx config from the Gitlab Nginx config.
 #
+# [*roles*]
```

Comment:
docs! \o/

Reply:
see my comment above. The standard for this module is to document something that is _not_ a feature of the omnibus package configuration. Documenting every config would be redundant, and could cause unexpected problems when gitlab changes the behavior or possible sub-key values.

There fore, the documentation provided for this param is adequate and in keeping with the standards of this module.